src/components/Settings/MobileTab.react.js

```diff
@@ -20,6 +20,47 @@ import MenuItem from '@material-ui/core/MenuItem';
 import { sortCountryLexographical } from '../../utils/helperFunctions';
 import { isPhoneNumber } from '../../utils';
 import { setUserSettings } from '../../apis';
+import styled from 'styled-components';
+
+const MobileDescription = styled.div`
+  margin-top: 0px;
```

Comment:
Not required, remove margin changes and make it inline style

Reply:
@fragm3 Made the changes.